packages/app-root/package.json

```diff
@@ -1,18 +1,19 @@
 {
-  "name": "@zooniverse/app-root",
+  "name": "@zooniverse/fe-root",
```

Comment:
Renaming app-root to fe-root is not necessary. I changed it to be consistent with the naming convention used with app-content-pages => fe-content-pages and app-project => fe-project, but if the folder name `app-root` and repo name `app-root` is preferred it would be a quick change back.